repository-data/application/src/main/resources/hcm-config/namespaces/hee/blockLinksReference.yaml

```diff
@@ -0,0 +1,92 @@
+definitions:
+  config:
+    /hippo:namespaces/hee/blockLinksReference:
+      jcr:primaryType: hipposysedit:templatetype
+      jcr:mixinTypes: ['editor:editable', 'mix:referenceable']
+      jcr:uuid: 6ec6d9de-c784-4219-aa30-43c7eea8bbfc
+      /hipposysedit:nodetype:
+        jcr:primaryType: hippo:handle
+        jcr:mixinTypes: ['mix:referenceable']
+        jcr:uuid: cae32b9f-bd6a-4bd8-97be-fa408bb2ab3d
+        /hipposysedit:nodetype:
+          jcr:primaryType: hipposysedit:nodetype
+          jcr:mixinTypes: ['mix:referenceable', 'hipposysedit:remodel']
+          jcr:uuid: ba767517-1050-4f29-837f-987b4a3ad3b1
+          hipposysedit:node: true
+          hipposysedit:supertype: ['hippo:compound', 'hippostd:relaxed']
+          hipposysedit:uri: http://www.heecmsplatform.com/hee/nt/1.0
+          /blockLinksContentBlock:
+            jcr:primaryType: hipposysedit:field
+            hipposysedit:mandatory: false
+            hipposysedit:multiple: false
+            hipposysedit:ordered: false
+            hipposysedit:path: hee:blockLinksContentBlock
+            hipposysedit:primary: false
+            hipposysedit:type: hippo:mirror
+            hipposysedit:validators: [required]
+          /blockLinksTitleOverride:
+            jcr:primaryType: hipposysedit:field
+            hipposysedit:mandatory: false
+            hipposysedit:multiple: false
+            hipposysedit:ordered: false
+            hipposysedit:path: hee:titleOverride
+            hipposysedit:primary: false
+            hipposysedit:type: Text
+            hipposysedit:validators: [optional]
+          /blockLinksDescriptionOverride:
```

Comment:
Could we change `blockLinksDescriptionOverride` as a `text` type field with `TextArea` editor (in order to be inline with other cards/blocks) ?

Reply:
Removed overrides for now.